dockerfile/anaconda-rpm/Dockerfile

```diff
@@ -22,13 +25,16 @@ LABEL maintainer=anaconda-devel@lists.fedoraproject.org
 # On ELN, BaseOS+AppStream don't have all our build dependencies; this provides the "Everything" compose
 COPY ["eln.repo", "/etc/yum.repos.d"]
 
+# The anaconda.spec.in is in the repository root. This file will be copied automatically here if
+# the build is invoked by Makefile.
+COPY ["anaconda.spec.in", "/root/"]
```

Comment:
Why is the file in the `/root` directory? We don't need it after the container build. Couldn't it be in `/tmp`?

Reply:
That is `/root` in the container, so it does not matter... much...

Reply:
Yep, it's just _some_ folder. We can go with `/tmp` but the problem is that `COPY` is happening in a separate layer than the `RUN` where it's used. I don't want to use `/tmp` because I'm worried that sometimes (maybe even here - not tested) the `/tmp` could be tmpfs and then the content could be missing for the `RUN` phase. I would rather like to avoid using `/tmp/` during container builds if it's not used during the same `RUN` phase.